docs/src/app_dev/daml_api/token_standard.rst

```diff
@@ -0,0 +1,276 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _token_standard_usage:
+
+Wallet Integration with Token Standard Assets
+=============================================
+
+This page provides wallet developers with guidance on how to integrate with token standard assets.
+Such an integration works by sending the right read and write requests to the Ledger API of the validator node hosting the wallet user's party.
+There are three kinds of integration patterns:
+
+  * :ref:`token_standard_usage_reading_contracts`
+  * :ref:`token_standard_usage_reading_tx_history`
+  * :ref:`token_standard_usage_executing_factory_choice`
+  * :ref:`token_standard_usage_executing_factory_choice`
+
+All of these integration patterns are demonstrated in the form of executable code as part of the experimental command-line interface for token standard assets.
+The sections below explaining the patterns below thus all start with a link to the code.
+They then provide additional context for an implementor.
+
+You can find the `ledger's OpenAPI definition <https://github.com/digital-asset/canton/blob/f608ec2cbb7b3e9331b7cc564eb260916606d815/community/ledger/ledger-json-api/src/test/resources/json-api-docs/openapi.yaml#L1#L1>`_.
+This is also accessible at ``http(s)://${YOUR_PARTICIPANT}/docs/openapi``.
+We encourage developers to use OpenAPI code generation tools as opposed to manually writing HTTP requests.
+
+Check out the :ref:`Authentication section <app-auth>` for more information on how to authenticate the requests.
```

Comment:
Why are we referring to this here? Does Ledger API auth work exactly the same as authentication to the splice apps, which is explained on that page?

If yes, then can you adjust that page to talk about both APIs?